pkgs/development/python-modules/pync/default.nix

```
@@ -9,14 +9,14 @@
 buildPythonPackage rec {
   version  = "2.0.3";
   pname = "pync";
-  disabled = ! isPy27;
 
   src = fetchPypi {
     inherit pname version;
     sha256 = "38b9e61735a3161f9211a5773c5f5ea698f36af4ff7f77fa03e8d1ff0caa117f";
   };
 
   buildInputs = [ pkgs.coreutils ];
+  checkInputs = [ pkgs.which ];
```

referencing pkgs is highly discourage, as it makes overriding very difficult, do you mind doing:
```suggestion
  nativeBuildInputs = [ coreutils ];
  checkInputs = [ which ];
```

and in `python-packages.nix`
```
pync = callPackage ../development/python-modules/pync {
  inherit (pkgs) coreutils which;
};
```